config/config_test.go

```diff
@@ -0,0 +1,53 @@
+package config_test
+
+import (
+	"os"
+
+	. "github.com/tidepool-org/platform/config"
```

Comment:
If we are going to use a different package (config_test), then I think we should NOT use the dot import. We want to show we are explicitly testing the `config` package. While I think the dot imports make a limited sense for ginkgo and gomega, I think they should be strongly avoided for our code.  For example, it isn't immediately clear that the code:

```
err := FromJSON(&random, "random.json")
```

is actually testing the config package.  However, if it were:

```
err := config.FromJSON(&random, "random.json")
```

then it would.